### PR TITLE
Cleanup packages/ember-metal/lib/properties.js

### PR DESCRIPTION
**Pull request by cibernox**

### CODE REVIEW

**Comment by stefanpenner** [comment]
restarting sauce

**Review by stefanpenner — APPROVED** [review]

**Comment by stefanpenner** [comment]
restarting sauce

File: packages/ember-metal/lib/properties.js [hunk 1]
```diff
@@ -58,7 +58,9 @@ export function MANDATORY_SETTER_FUNCTION(name) {
 export function DEFAULT_GETTER_FUNCTION(name) {
   return function GETTER_FUNCTION() {
     let meta = peekMeta(this);
-    return meta && meta.peekValues(name);
+    if (meta !== undefined) {
```
Comment (bekzod) [inline packages/ember-metal/lib/properties.js]: I think meta can be null too

File: packages/ember-metal/lib/properties.js [hunk 2]
```diff
@@ -67,7 +69,10 @@ import { UNDEFINED } from './meta';
 export function INHERITING_GETTER_FUNCTION(name) {
   function IGETTER_FUNCTION() {
     let meta = peekMeta(this);
-    let val = meta && meta.readInheritedValue('values', name);
+    let val;
+    if (meta !== undefined) {
```
Comment (bekzod) [inline packages/ember-metal/lib/properties.js]: I think meta can be null too

Comment (cibernox) [inline packages/ember-metal/lib/properties.js]: You're right. Both fixed

**Pull request merged**